src/Browser.cpp

```diff
@@ -43,7 +43,7 @@ extern "C" {
 String Browser::currentDir;
 bool Browser::qwertyVisible;
 
-CStringArray Browser::fileItems(sizeof(FileItem));
+FileItemArray Browser::fileItems(sizeof(FileItem));
```

Comment:
Assuming my grep skills are OK (which given the hour I'm typing this at they may not be) this was the only thing using `CStringArray`. If you want to rename that type instead of copying it that may not be the worst idea.